src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1984,8 +1984,13 @@ void searchContextProviders
 * 0. Preparations
 * 1. Preconditions
 * 2. Get the complete list of entities from mongo
+*
+* RETURN VALUE
```

Comment:
If the return value is just to distinguish between "success" or "fail", maybe is better to use a bool for this? I think the style in mongoBackend is using bool for sucess/fail returns.